.github/workflows/linting-and-tests.yml

```diff
@@ -408,8 +423,7 @@ jobs:
           GRAFANA_PASSWORD: oncall
           MAILSLURP_API_KEY: ${{ secrets.MAILSLURP_API_KEY }}
         working-directory: ./grafana-plugin
-        # -x = exit command after first failing test
-        run: yarn test:integration -x
```

Comment:
don't exit on first failed test as this defeats the purpose of allowing the retries to get around tests which are currently flaky